src/runtime/virtcontainers/clh.go

```diff
@@ -638,10 +638,6 @@ func (clh *cloudHypervisor) HotplugAddDevice(ctx context.Context, devInfo interf
 	span, _ := katatrace.Trace(ctx, clh.Logger(), "HotplugAddDevice", clhTracingTags, map[string]string{"sandbox_id": clh.id})
 	defer span.End()
 
-	if clh.config.ConfidentialGuest {
-		return nil, errors.New("Device hotplug addition is not supported in confidential mode")
-	}
-
```

Comment:
My fault for mentioning that device hot plug is not supported with TDX. Even though virtio device hot plug is supported, I don't think passthrough is. As a matter of fact, I don't think you can do passthrough at all with neither SEV nor TDX.

Reply:
I'd like to cover this in a different PR, if you don't mind.